pkg/leeway/signing/attestation.go

```diff
@@ -0,0 +1,365 @@
+package signing
+
+import (
+	"context"
+	"crypto/sha256"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/in-toto/in-toto-golang/in_toto"
+	"github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/common"
+	slsa "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v0.2"
+	"github.com/sigstore/sigstore-go/pkg/root"
+	"github.com/sigstore/sigstore-go/pkg/sign"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// GitHubContext contains GitHub Actions environment information
+type GitHubContext struct {
+	RunID        string // GITHUB_RUN_ID
```

Comment:
FYI we use [`go-simpler.org/env`](https://github.com/go-simpler/env) in Ona all over the place, to the point where I thought it was part of the stdlib... :sweat_smile: 

But we don't have to add that dependency here/now. :+1: